Certified-associate-in-project-management-domain1.md

```
@@ -0,0 +1,548 @@
+    style C fill:#7c5cff,stroke:#5a3fd6,color:#ffffff
+    style R1 fill:#2f9e8f,stroke:#1f6f65,color:#ffffff
+    style R2 fill:#2f9e8f,stroke:#1f6f65,color:#ffffff
+    style R3 fill:#2f9e8f,stroke:#1f6f65,color:#ffffff
+    style R4 fill:#2f9e8f,stroke:#1f6f65,color:#ffffff
+```
+
+**ベストプラクティス**
+- 試験ではシナリオ形式で「あなたならどう行動するか」を問う設問が頻出する。判断の基準は「個人的な利害」ではなく「4つの価値観、特に公正・誠実」に照らして最も倫理的な行動を選ぶこと。
+- 利益相反(Conflict of Interest)が疑われる場合は、まず開示(disclose)することが原則。
+
+出典: PMI Code of Ethics and Professional Conduct(公式PDF)[3]、PMI Ethics Guidelines [4]
```

markdownlint-cli2 (0.23.2)

[warning] Reversed link syntax
(公式PDF)[3]

(MD011, no-reversed-links)